app/src/test/java/com/concordia/mcga/adapters/BuildingInfoAdapterTest.java

```diff
@@ -0,0 +1,116 @@
+package com.concordia.mcga.adapters;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.ListView;
+import android.widget.TextView;
+
+import com.concordia.mcga.activities.BuildConfig;
+import com.concordia.mcga.activities.R;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.annotation.Config;
+import org.robolectric.res.FileTypedResource;
+
+import java.util.ArrayList;
+import static junit.framework.Assert.assertNotNull;
+import static junit.framework.Assert.assertNull;
+import static junit.framework.Assert.assertTrue;
+import static org.mockito.Mockito.when;
+
+
+@RunWith(RobolectricTestRunner.class)
+@Config(constants = BuildConfig.class,sdk = 21)
```

Comment:
does the SDK have to be specified here? I suspect it's because of that its failing